packages/affiliates/test/models.js

```diff
@@ -74,8 +74,10 @@ describe("History", function() {
 describe("Prices", function() {
   let prices, seed;
   const token = "0xD16c79c8A39D44B2F3eB45D2019cd6A42B03E2A9";
+  const startingTimestamp = moment("2020-09-23 23:00:00", "YYYY-MM-DD  HH:mm Z").valueOf();
+  const endingTimestamp = moment("2020-10-05 23:00:00", "YYYY-MM-DD  HH:mm Z").valueOf();
   it("init", async function() {
-    seed = await Coingecko().chart(token, "usd", "10");
+    seed = await Coingecko().chart(token, "usd", startingTimestamp, endingTimestamp);
```

Comment:
i forot about this test, but its making a real call out to coingecko in CI. seems fine to me but what are your thoughts on that?

Reply:
I think it's fine for now. we should refactor it to use the sample set though in the future.

Reply:
We should probably mock the API call instead of making the real one. We've had previous tests break that used to call out to Intrinio API if you recall.